### PR TITLE
[v2.7.1] Add the cluster-id to log messages during cluster creation

### PR DESCRIPTION
**Pull request by sbstp**
## Issue: 
#21637 
 
## Problem

Add the cluster-id to log messages during cluster creation in order to make trouble shooting easier.
 
## Solution

Add the Node's cluster id to the logs when forwarding messages coming from the driver.
 
### Manual Testing

1. Create a new cluster with the UI and look at the logs produced in stdout/stderr.

### CODE REVIEW

File: pkg/controllers/management/node/utils.go
Comment (HarrisonWAffel) [inline pkg/controllers/management/node/utils.go:228]: We probably also want to include the new prefix here, for consistency

File: pkg/controllers/management/node/utils.go
Comment (HarrisonWAffel) [inline pkg/controllers/management/node/utils.go:230]: I think you might have left `m.clusterManager.` in accidentally

**Review by HarrisonWAffel — CHANGES_REQUESTED** [review]

File: pkg/controllers/management/node/utils.go
Comment (sbstp) [reply pkg/controllers/management/node/utils.go]: Indeed. I forgot to add the fix.

File: pkg/controllers/management/node/utils.go
Comment (sbstp) [reply pkg/controllers/management/node/utils.go:228]: Yeah that makes sense. Updated.

**Review by HarrisonWAffel — APPROVED** [review]
LGTM, this will add a prefix to all rancher-machine logs that are produced when creating RKE1 clusters. This still needs one more review from team 2, but should not be merged until v2.7.0 has been released.

**Comment by sbstp** [comment]
> but should not be merged until v2.7.0 has been released.

Should I put the version in the title to v2.7.1 then?

**Comment by HarrisonWAffel** [comment]
@sbstp Yeah that's a good idea. Also, adding `DNM` to issues that should not be merged is good practice, you can always update the title later

**Comment by sbstp** [comment]
Just to be clear, the reason you think it shouldn't go into 2.7.0 is because it's really close to release?

**Comment by snasovich** [comment]
> Just to be clear, the reason you think it shouldn't go into 2.7.0 is because it's really close to release?

Yes, we're well past "bugfix complete" date and are only addressing critical release issues now.

**Comment by sbstp** [comment]
I wasn't able to find a do not merge / DMN label in github labels.

**Comment by snasovich** [comment]
> I wasn't able to find a do not merge / DMN label in github labels.

@sbstp , there is not a formal label for DNM, we're putting it in title, e.g. like `[DNM - v2.7.1] PR title`.

**Comment by snasovich** [comment]
@HarrisonWAffel @sbstp , as mentioned above, this addresses prefixing rancher-machine messages with cluster ID for RKE1 provisioned clusters.
I know the issue is about RKE1, but when it was opened there was no RKE2 provisioning.
Do we already have cluster ID included for rancher-machine messages generated during RKE2 clusters provisioning? If not, what would it take to cover this case as well?

**Comment by HarrisonWAffel** [comment]
@snasovich When provisioning RKE2/K3s clusters Rancher will create a job to run the rancher-machine binary in a container. The logs of that container are not piped to the UI or to the rancher server logs. The naming of this already job denotes the cluster and node pool the new node belongs to, using the following format 

`job.batch/<CLUSTER_NAME>-<POOL_NAME>-<MGMNT_CLUSTER_ID>-<MACHINE_ID>-machine-provision`, e.g.
`job.batch/haffel-test-2-pool1-71cf79c1-rmg56-machine-provision`

**Review by snasovich — APPROVED** [review]

**Pull request merged**
